Source/WebCore/animation/ScrollTimeline.cpp

```diff
@@ -31,6 +31,8 @@
 #include "CSSScrollValue.h"
 #include "CSSValuePool.h"
 #include "Element.h"
+#include "RenderLayerScrollableArea.h"
```

Comment:
Do you need either of these includes now?

Reply:
These are still needed.